src/Appwrite/Utopia/Database/Validator/Queries/Executions.php

```diff
@@ -7,7 +7,7 @@ class Executions extends Base
     public const ALLOWED_ATTRIBUTES = [
         'trigger',
         'status',
-        'statusCode',
+        'responseStatusCode',
```

Comment:
could you remind me why we changed the attribute name ?

Reply:
Because we now also have requestStatusCode. We use prefix `request` and `response` for all HTTP-related attributes (`requestMethod`, `responseHeaders`, ..)